### PR TITLE
Show more documents

### PR DESCRIPTION
**Pull request by agnessa**
Implements the "show more documents" functionality in the document search results as well as the documents tab on species pages. The back-end implementation has been changed to facilitate loading each document group (EC SRG, CITES CoP, ...) separately, which allowed to simplify the serializers. "Sortable columns" is now implemented in back-end.

The admin search now uses the same faster search mechanism as the public one (sans the language grouping and document access level checks), as the old admin mechanism was not scaling.

Requires `rake db:migrate` and `Rails.cache.clear` (changes to low-level cache of search results)

### CODE REVIEW

File: app/serializers/species/document_serializer.rb [hunk 1]
```diff
@@ -11,13 +11,13 @@ def document_type
...
+  # def taxon_names
+  #   object.taxon_names && parse_pg_array(object.taxon_names) || []
+  # end
 
-  def geo_entity_names
-    object.geo_entity_names && parse_pg_array(object.geo_entity_names) || []
-  end
+  # def geo_entity_names
+  #   object.geo_entity_names && parse_pg_array(object.geo_entity_names) || []
+  # end
```
Comment (Levia) [inline app/serializers/species/document_serializer.rb]: Can this commented code be removed completely?

Comment (agnessa) [inline app/serializers/species/document_serializer.rb]: of course, thank you!

File: app/assets/stylesheets/species/all.css [hunk 2]
```diff
@@ -1087,12 +1087,15 @@ html.lt-ie9 #main .tabset > li > ul >li.first {
...
 #main .block thead tr { background: #f5f5f7; }
-#main .block.show_more .btn-hloder {
+.block.show_more .btn-hloder {
 	margin: 0;
 	text-align: center;
   display: block;
 }
-#main .block.show_more:hover .btn-hloder {
+.inner-table-container.show_more .btn-hloder {
+  margin: 0 0 -14px 0;
```
Comment (Levia) [inline app/assets/stylesheets/species/all.css]: I find the following values to make it look better:
margin: -10px 0 -2px 0;
But it's my personal preference :)

Comment (agnessa) [inline app/assets/stylesheets/species/all.css]: yes, makes sense :)

**Pull request merged**